src/av/errors.rs

```diff
@@ -0,0 +1,286 @@
+use std::error::Error;
+use std::fmt;
+
+///////////////////////////////
+// Creation and destruction //
+/////////////////////////////
+/// Creation and destruction
+#[repr(C)]
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum NewError {
```

Comment:
This path rstox::av::errors::NewError, not confusion. Of course, I am willing to put it into NewAvError.